cms/methodology/tests/factories.py

```diff
@@ -0,0 +1,47 @@
+from datetime import timedelta
+from typing import ClassVar
+
+import factory
+import wagtail_factories
+from django.utils import timezone
+
+from cms.core.tests.factories import ContactDetailsFactory, RichTextBlockFactory
+from cms.methodology.models import MethodologyPage
+from cms.topics.tests.factories import TopicPageFactory
+
+
+class SectionContentBlockFactory(wagtail_factories.StructBlockFactory):
+    """Factory for Section content block."""
+
+    title = factory.Faker("text", max_nb_chars=50)
+    content = wagtail_factories.StreamFieldFactory(
+        {
+            "rich_text": factory.SubFactory(RichTextBlockFactory),
+        }
+    )
+
+
+class SectionBlockFactory(wagtail_factories.StructBlockFactory):
+    """Factory for Section block."""
+
+    title = factory.Faker("text", max_nb_chars=50)
+    content = factory.SubFactory(SectionContentBlockFactory)
```

Comment:
I think the previous comment got lost. These two belong in core/tests/factories, now that the blocks are in the core app